client/src/index.css

```diff
@@ -1239,4 +1278,17 @@ div[id*='fg-entityAssessment'] {
   .form-horizontal .form-label {
     text-align: left;
   }
+  .event-collection header {
+    margin-bottom: 18px;
+  }
+
+  .event-collection footer {
+    margin-top: 18px;
+  }
+
+  .event-summary {
+    margin-top: 25px;
+    padding-top: 15px;
+    box-shadow: 0 1px 3px hsla(0, 0%, 0%, 0.15);
+  }
```

Comment:
This appears to be the exact same styling already present in the main stylesheet above. Why is this here as well?